### PR TITLE
Allow read-only access to pulp view repositories/ansible/ansible

### PR DESCRIPTION
**Pull request by awcrosby**
#### What is this PR doing:

The default permissions for pulp view repositories/ansible/ansible does not allow read access for non-admin users. This fixes that.

We now use this endpoint for collection singing: https://github.com/ansible/galaxy_ng/blob/0fc068c926321e2052dd8f0621ff9b54e302bc66/galaxy_ng/app/access_control/access_policy.py#L144

No-Issue

#### Notes: 

**PR Author**: Add a QE reviewer ([exceptions](https://docs.engineering.redhat.com/display/AUTOHUB/Other+Team+Processes#OtherTeamProcesses-galaxy_ngrepo)); 
**Reviewers**: look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

### CODE REVIEW

File: .github/workflows/scripts/before_install.sh [hunk 1]
```diff
@@ -122,7 +122,7 @@ fi
 cd ..
 
 
-git clone --depth=1 https://github.com/pulp/pulp_ansible.git --branch 0.13
+git clone --depth=1 https://github.com/pulp/pulp_ansible.git --branch 0.13.0
```
Comment (alikins) [inline .github/workflows/scripts/before_install.sh:125]: Should the CI tooling related changes go into a separate PR?

File: template_config.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 additional_repos:
 - bindings: true
-  branch: 0.13
+  branch: 0.13.0
```
Comment (alikins) [inline template_config.yml:8]: Should this also be separate pr?

File: galaxy_ng/tests/integration/api/test_pulp_api.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import pytest
+from ansible.galaxy.api import GalaxyError
+
+from ..utils import get_client
+
+
+@pytest.mark.standalone_only
+@pytest.mark.pulp_api
+def test_pulp_api_redirect(ansible_config, artifact):
```
Comment (alikins) [inline galaxy_ng/tests/integration/api/test_pulp_api.py:9]: This seems out of place with the rest of the PR to me. Useful test, not sure it makes sense in this PR

**Review by alikins — CHANGES_REQUESTED** [review]
Does this pr need to be split up?

File: .github/workflows/scripts/before_install.sh (hunk 1 above)
Comment (awcrosby) [reply .github/workflows/scripts/before_install.sh]: moved to https://github.com/ansible/galaxy_ng/pull/1307

**Review by newswangerd — APPROVED** [review]

File: galaxy_ng/tests/integration/api/test_pulp_api.py (hunk 3 above)
Comment (alikins) [reply galaxy_ng/tests/integration/api/test_pulp_api.py:9]: ah

**Review by alikins — APPROVED** [review]
Not sure why I still see the commit 0cf492ad165cfbe618f8141eaf2d523f8fa94c53 in this pr, but the rest looks okay to.

**Review by rochacbruno — APPROVED** [review]
